### PR TITLE
lkl: add LKL_HIJACK_NET_QDISC to configure qdisc policy

### PR DESCRIPTION
**Pull request by thehajime**
Also enable SCH_FQ to be able to use it.

Signed-off-by: Hajime Tazaki <thehajime@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/lkl/linux/315)

### CODE REVIEW

File: Documentation/lkl.txt [hunk 1]
```diff
@@ -202,6 +202,12 @@ are the list of those variable for your environment.
      $ LKL_HIJACK_NET_NEIGHBOR="192.168.13.100|12:34:56:78:9a:bc;2001:db8:0:f101::3|12:34:56:78:9a:be"
      lkl-hijack.sh ip neighbor show
 ```
+* LKL_HIJACK_NET_QDISC
+
+  Add a qdisc entry in the form of "ip|mac;ip|mac;...".
```
Comment (liuyuan10) [inline Documentation/lkl.txt]: ip|mac?

File: tools/lkl/include/lkl.h [hunk 2]
```diff
@@ -461,6 +461,24 @@ int lkl_if_wait_ipv6_dad(int ifindex, void *addr);
...
+int lkl_qdisc_add(int ifindex, char *root, char *type);
+
+/**
+ * lkl_qdisc_parse_add - Add a qdisc entry for an interface with strings
+ *
+ * @ifindex - the ifindex of the interface
+ * @entries - strings of qdisc configurations in the form of
+ *            "root|type;root|type;..."
+ */
+void lkl_qdisc_parse_add(int ifindex, char* entries);
```
Comment (liuyuan10) [inline tools/lkl/include/lkl.h]: I don't think this should be part of lkl.h.

**Comment by thehajime** [comment]
> There is a small coding style issue, but the code looks good.

Ah thanks,  I found those issues.  will fix them.

I was wondering why circleci didn't report those though..

File: Documentation/lkl.txt (hunk 1 above)
Comment (thehajime) [inline Documentation/lkl.txt]: Thanks, will fix this.

**Comment by thehajime** [comment]
> Can it be added to test?

good catch - I will add a test.

File: tools/lkl/include/lkl.h (hunk 2 above)
Comment (thehajime) [inline tools/lkl/include/lkl.h]: the same reason as https://github.com/lkl/linux/pull/319#discussion_r100242030
I believe those are useful for others.

maybe other LKL_HIJACK_xxx configs (arp, nd, etc) can be moved in this sense.

**Comment by thehajime** [comment]
> I was wondering why circleci didn't report those though..

Off topic: I found I configured SKIP_CHECKPATCH in my circleci instance and this prevents me to check the style in advance. So everything is fine.

**Pull request merged**